apps/docs/package.json

```diff
@@ -14,10 +14,10 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@fuel-ts/versions": "workspace:*",
     "@fuel-ts/errors": "workspace:*",
+    "@fuel-ts/versions": "workspace:*",
```

Comment:
I've left this comment here to avoid any merges.

Although I have tested this locally and it works great with the docs hub. With consideration of the number of breaking changes in the release of [`0.94.0`](https://github.com/FuelLabs/fuels-ts/pull/2863) - I would advise we hold off until `0.95.0` for this change.

Reply:
Thanks for testing locally. I agree, in the past the docs hub integration has been pretty fragile. Let's do this in isolation, could even fast follow `0.94.0`.

Reply:
Could we get this one across now that `0.94` is out?